python/python/tests/tests_server_modules/search/test_ft_create.py

```diff
@@ -63,7 +63,9 @@ async def test_ft_create(self, glide_client: GlideClusterClient):
             name="vec",
             algorithm=VectorAlgorithm.HNSW,
             attributes=VectorFieldAttributesHnsw(
-                dim=2, distance_metric=DistanceMetricType.L2, type=VectorType.FLOAT32
+                dimensions=2,
+                distance_metric=DistanceMetricType.L2,
+                type=VectorType.FLOAT32,
```

Comment:
I made vector type hardcoded in java impl. Should we define an enum with 1 value in java to let user use it?

Reply:
I don't care.  
It's harder to make optional in Java. So let's just leave it as is...